attorch/losses.py

```diff
@@ -32,7 +32,7 @@ def forward(self, output, target):
 
 
 class Corr(nn.Module):
-    def __init__(self, eps=1e-14):
+    def __init__(self, eps=1e-10):
```

Comment:
what about the epsilon of the other losses. Maybe everything should be set up to 1e-10 or 1e-8 for consistency. There's also another eps set to 1e-6 as default in l.121 of layers.py